src/main/java/com/endava/algoritm/Solution.java

```diff
@@ -0,0 +1,33 @@
+package com.endava.algoritm;
+
+public class Solution {
+
+    public static int solution(int[] A) {
+        int minMin = Integer.MAX_VALUE;
+        int min = Integer.MAX_VALUE;
+        int max = Integer.MIN_VALUE;
+        int maxMax = Integer.MIN_VALUE;
+        int maxMaxMax = Integer.MIN_VALUE;
```

Comment:
what would be better naming convention here?
maxMax, maxMaxMax, minMin are not descriptive and someone reading the code would have no idea what is their purpose.

Reply:
I do not know how to name it :( First it was min1, min2... then I realized it is confusing for me. Any suggestions are appreciated :)

Reply:
Can you describe in a sentence or two, what the use for maxMax and what is the use for maxMaxMax?

Reply:
The more **max**'s there are in the name, number is bigger.
The more **min**'s there are in the name, number is smaller. 
:)

Maybe to add comment like this:
```
// Order from smaller to bigger:
// minMin <= min <= max <= maxMax <= maxMaxMax
```

Reply:
I don't like comments, but that's just my code style. Comment is valid option here.
If i wrote the code i would name them: max, secondMax ...
secondMax clearly tells me that that's second biggest number;

Reply:
Make sense. Code changed.